containers/app/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN npm install -g npm@10.5.1
 RUN npm ci
 
 COPY ./frontend ./
-RUN npm run make-i18n && npm run build
```

Comment:
do we not need make-i18n anymore?

Reply:
`npm run build` already runs `npm run make i18n`

https://github.com/All-Hands-AI/OpenHands/blob/2277897f863a046914dec20bbf36e75cfd081a45/frontend/package.json#L48